lib/main.js

```diff
@@ -339,7 +339,9 @@ function ConfigureMathJax() {
         delete STYLES[".mjx-chartest .mjx-box"];
         delete STYLES[".mjx-test"];
         delete STYLES[".mjx-ex-boxtest"];
-
+        //  fontURL to current MathJax version
+        if (!fontURL)
+        fontURL = 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/' + MathJax.version + '/fonts/HTML-CSS';
```

Comment:
Probably should be indented, if not braced (Volker would vote for braces).

Reply:
Oh, that should have been on the same line (otherwise we overwrite a custom fontURL, no?)

Reply:
No, it is still part of the `if` statement, even without braces.  The newline doesn't change the meaning.  Since it is a long expression, having it on a different line is fine.  Using braces would not be a bad idea (and will make Volker happy).

Reply:
> will make @zorkow happy

Now how could I refuse that :smile:

Reply:
oh, I apparently did not know that.